src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3795,6 +3795,10 @@ spread.ElementPlaceholder is { } elementPlaceholder &&
                         VisitRvalue(keyValuePair.Value);
                         // PROTOTYPE: Check nullability from conversions of key and value.
                         break;
+                    case BoundIndexerAssignmentFromExpressionElement indexerAssignment:
```

Comment:
Do we need to track some follow-up changes to ref analysis too, to handle the new nodes?